go.mod

```diff
@@ -92,3 +92,8 @@ require (
 	go.elastic.co/go-licence-detector v0.6.0
 	gopkg.in/yaml.v2 v2.4.0
 )
+
+retract (
+	v1.0.1 // Contains retractions only.
```

Comment:
I'm just a bit confused why also retracting this one. What is the plan for releasing this? Would we release `v1.0.1` to retract `v1.0.0` and then another `v0.x.y` to have the latest being in the `v0.x` range?